lib/airbrake-ruby/filters/thread_filter.rb

```diff
@@ -57,6 +64,7 @@ def thread_variables(th)
 
       def fiber_variables(th)
         th.keys.map.with_object({}) do |key, h|
+          next if IGNORED_FIBER_VARIABLES.any? { |v| v == key }
```

Comment:
Can we ignore all vars that starts with `_`? I wanted to suggest this before, but did not want to complicate without a good reason.